cypress/e2e/ModelizerPage/SwitchView/add.component.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Test switch model to text view: add component/link
+
+  Background:
+    Given I clear cache
+    And I set viewport size to "1536" px for width and "960" px for height
+    And I visit the "/"
+
+    When I click on "[data-cy=\"new-project\"]"
+    Then I expect current url is "/modelizer/project-[a-f0-9]{8}/model"
+    And  I extract "project-[a-f0-9]{8}" from url in field "projectName" of context
+    And  I expect "[data-cy=\"plugin-definitions-leto-modelizer-plugin-test\"]" exists
+    And  I expect "[data-cy=\"plugin-definitions-leto-modelizer-plugin-test\"] [data-cy=\"plugin-definitions-title\"]" is "leto-modelizer-plugin-test"
+    
+    When I click on "[data-cy=\"plugin-definitions-leto-modelizer-plugin-test\"]"
+    Then I expect ".plugin-definitions" appear 1 time on screen
+    And  I expect ".component-definition-card" appear 7 times on screen
+
+  Scenario: Add a component (Model view) should create plugin and project configuration files (Text view)
```

Comment:
create  project configuration files and plugin files

Reply:
done